### ISSUE TITLE
support SPDX composite license expressions

### ISSUE THREAD

**Issue by sdavids** [hadolint/hadolint#992]
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

No warning.

### Actual behavior

```
DL3054 warning: Label `org.opencontainers.image.licenses` is not a valid SPDX identifier.
```

### Steps to reproduce the behavior

```shell
$ hadolint --version
Haskell Dockerfile Linter 2.12.0
```

**.hadolint.yaml**
```yml
label-schema:
  org.opencontainers.image.licenses: spdx
```

**Dockerfile**

```Dockerfile
LABEL org.opencontainers.image.licenses="Apache-2.0 OR MIT" 
```

**References**

- [Open Container Initiative - Pre-Defined Annotation Keys](https://github.com/opencontainers/image-spec/blob/main/annotations.md)
- [SPDX composite license expressions](https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/#d4-composite-license-expressions)
